specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/examples/customApisCreateOrUpdate.json

```diff
@@ -0,0 +1,55 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testResourceGroup",
+    "apiName": "testCustomApi",
+    "api-version": "2016-06-01",
+    "customApi": {
+      "properties": {
+        "runtimeUrls": [
+          "https://tempuri.org",
+          "http://tempuri.org"
+        ],
+        "capabilities": [],
+        "description": "",
+        "displayName": "testCustomApi",
+        "iconUri": "/testIcon.svg",
+        "apiDefinitions": {
```

Comment:
PUT request has the swagger inline (property name is "swagger")

Reply:
Added, currently with an empty body... I'm not sure if we should include an actual sample swagger spec in there. It seems like that could get out of hand fast.

Reply:
Empty is fine